### PR TITLE
minor fix & update

### PR DESCRIPTION
**Pull request by dexhunter**
* start working on #14 & #94
* update command
* add ignore file

Signed-off-by: Dixing (Dex) Xu <dixingxu@gmail.com>

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -5,6 +5,7 @@
 *.so
 *.dylib
 ./tape
+tape
```
Comment (SamYuan1990) [inline .gitignore:8]: no need?

File: README.md [hunk 2]
```diff
@@ -43,7 +43,7 @@ You could get `tape` in three ways:
 
 #### Binary
 
-Execute `./tape -c config.yaml -n 40000` to generate 40000 transactions to Fabric.
+Execute `./tape run -c config.yaml -n 40000` to generate 40000 transactions to Fabric.
```
Comment (SamYuan1990) [inline README.md:46]: why run?

**Review by SamYuan1990 — CHANGES_REQUESTED** [review]

File: .gitignore (hunk 1 above)
Comment (SamYuan1990) [reply .gitignore:8]: em... or this is just t avoid tape binary here?

File: README.md (hunk 2 above)
Comment (dexhunter) [reply README.md:46]: I think it's the default command as seen [there](https://github.com/Hyperledger-TWGC/tape/blob/master/cmd/tape/main.go#L21) besides if you print help the usage indicates the same

File: .gitignore (hunk 1 above)
Comment (dexhunter) [reply .gitignore:8]: after I execute `make tape` there is a binary there so I am just excluding this binary for uploading

Comment (SamYuan1990) [reply .gitignore:8]: I see.

**Comment by SamYuan1990** [comment]
@dexhunter , are you still working on this?

**Pull request closed without merging**